### PR TITLE
Submitting transfer learning tutorial with tf 2 and tf hub

### PR DESCRIPTION
**Pull request by padominguez**
Using TF 2.1.0 in this example, up to and including save_model.

### CODE REVIEW

File: notebooks/samples/tensorflow/transfer_learning/ai_platform_transfer_learning.ipynb [hunk 1]
```diff
@@ -0,0 +1,847 @@
+{
```
Comment (gogasca) [inline notebooks/samples/tensorflow/transfer_learning/ai_platform_transfer_learning.ipynb]: <p>Ok for now, later we need to update this to show how to model training and prediction in AI Platform</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (gogasca) [inline notebooks/samples/tensorflow/transfer_learning/ai_platform_transfer_learning.ipynb]: <p>We need TF 2.1.0, remove tf-nightly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

**Comment by gogasca** [comment]
@padominguez Overall looks good, minor comments, added. Once resolved I can Merge

File: notebooks/samples/tensorflow/transfer_learning/ai_platform_transfer_learning.ipynb (hunk 1 above)
Comment (padominguez) [reply notebooks/samples/tensorflow/transfer_learning/ai_platform_transfer_learning.ipynb]: Addressed this!

Comment (padominguez) [reply notebooks/samples/tensorflow/transfer_learning/ai_platform_transfer_learning.ipynb]: Got it; will be updating soon.

**Pull request merged**